book/B-embedding-git/sections/libgit2.asc

```diff
@@ -32,31 +32,28 @@ git_repository_free(repo);
 
 Ang pangalawang tipak ng code ay gumagamit ng rev-parse na syntax (tingnan ang <<_git_tools#_branch_references>> para sa higit pa nito) upang makuha ang commit na ang HEAD na sa kalaunan.
-Ang binalik na uri ay isang `git_object` pointer, na kumakatawan sa isang bagay na umiiral sa database ng Git object para sa isang repository.
-`git_object` is actually a ``parent'' type for several different kinds of objects; the memory layout for each of the ``child'' types is the same as for `git_object`, so you can safely cast to the right one.
-In this case, `git_object_type(commit)` would return `GIT_OBJ_COMMIT`, so it's safe to cast to a `git_commit` pointer.
+Ang binalik na uri ay isang `git_object` pointer, na kumakatawan sa isang bagay na umiiral sa database ng Git object para sa isang repository. `git_object` ay talagang isang ``parent'' na uri para sa maraming iba't ibang mga uri ng mga bagay; ang layout ng memorya para sa bawat isa sa mga uri ng ``child'' ay kapareho ng para sa `git_object`, para maaari mong isumite ng ligtas ang tama. Sa kasong ito, ang `git_object_type(commit)` ay babalik sa `GIT_OBJ_COMMIT`, kaya ligtas itong isumite sa isang `git_commit` pointer.
 
-The next chunk shows how to access the commit's properties.
-The last line here uses a `git_oid` type; this is Libgit2's representation for a SHA-1 hash.
+Ang susunod na bahagi ay nagpapakita kung paano ma-access ang mga katangian ng commit.
+Ang huling linya dito ay gumagamit ng isang uri ng `git_oid`; ito ang representasyon ng Libgit2 para sa SHA-1 hash.
 
-From this sample, a couple of patterns have started to emerge:
+Mula sa sample na ito, ang ilang mga anyo ay nagsimula na lumitaw:
 
-* If you declare a pointer and pass a reference to it into a Libgit2 call, that call will probably return an integer error code.
-  A `0` value indicates success; anything less is an error.
-* If Libgit2 populates a pointer for you, you're responsible for freeing it.
-* If Libgit2 returns a `const` pointer from a call, you don't have to free it, but it will become invalid when the object it belongs to is freed.
-* Writing C is a bit painful.
+* Kung ipinapahayag mo ang isang pointer at magpasa ng isang reperensiya nito sa isang Libgit2 call, ang tawag na iyon ay malamang na babalik ang isang error code ng integer.
+  Ang halaga na `0` ay nagpapahiwatig ng tagumpay; Ang anumang mas mababa ay mali.
+* Kung pinarami ang pointer ng Libgit2 para sa iyo, ikaw ay responsable para sa pagbabakanti nito.
+* Kung ang Libgit2 ay magbabalik ng isang `const` na pointer mula sa isang tawag, hindi mo kailangang palayain ito, ngunit magiging imbalido kapag ang bagay nabibilang nito ay napalaya.
```

Comment:
> kapag ang bagay na kinabibilangan nito ay napalaya

Reply:
Done changing it to "kinabibilangan". Thanks for the review @heywens